### DISCUSSION TITLE
Feature request - [Chinese support]

### DISCUSSION THREAD

**Discussion by lfxx** [snakers4/silero-vad#40] [Ideas]
Hope Chinese can be added into this excellent tool.

**Comment by snakers4** [comment]
We have an internal dependency on our STT and for now there are no plans for Chinese unless we sign some sort of contract / deal (we do not speak Chinese alas,  we would need at least one person with language / culture)

You are encouraged nevertheless to try the vad as is - there is a high probability it will just work out of the box (please report if it does)

If you will be trying it, please make sure you use a visualization tool to tune the params

We will be making the post processing simpler soon though

**Comment by lfxx** [comment]
> We have an internal dependency on our STT and for now there are no plans for Chinese unless we sign some sort of contract / deal (we do not speak Chinese alas, we would need at least one person with language / culture)
> 
> You are encouraged nevertheless to try the vad as is - there is a high probability it will just work out of the box (please report if it does)
> 
> If you will be trying it, please make sure you use a visualization tool to tune the params
> 
> We will be making the post processing simpler soon though

So this repo may work on Chinese now?if yes,i will try

**Comment by snakers4** [comment]
> So this repo may work on Chinese now?if yes,i will try

The vad may

Language classifier / number detector obviously won't

**Comment by lfxx** [comment]
> > So this repo may work on Chinese now?if yes,i will try
> 
> The vad may
> 
> Language classifier / number detector obviously won't

Got the result below:
[{'end': 290000, 'start': 36000}]
what's the meaning of 290000 and 36000?
Another question:
I do the vad using the function below:
speech_timestamps = get_speech_ts(wav, model, visualize_probs=True,num_steps=4)
but no chart appears when the scripts finished.Any suggestions?thanks!

**Comment by snakers4** [comment]
> what's the meaning of 290000 and 36000?

https://github.com/snakers4/silero-vad/discussions/25

> speech_timestamps = get_speech_ts(wav, model, visualize_probs=True,num_steps=4)
> but no chart appears when the scripts finished.Any suggestions?thanks!

this has to be run in a notebook
